letsencrypt/crypto_util.py

```diff
@@ -231,3 +231,16 @@ def make_ss_cert(key_str, domains, not_before=None,
     assert cert.verify()
     # print check_purpose(,0
     return cert.as_pem()
+
+
+def get_sans_from_cert(pem):
```

Comment:
This function should internally use `_request_san` as of #399 (pyOpenSSL). (Although `_request_san` should be renamed to something more generic, say `_pyopenssl_certificate_san`)